### PR TITLE
SW-5089 Add endpoint to list accelerator orgs

### PR DESCRIPTION
**Pull request by sgrimm**
`GET /api/v1/accelerator/organizations` returns a list of organizations with the
Accelerator internal tag and a list of each of those organizations' projects.
By default, it omits projects that are already assigned to participants.

Access depends on the user's ability to read internal tags, which is extended to
include TF Expert and Read Only global roles.

### CODE REVIEW

**Comment by sgrimm** [comment]
* **#1791** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1791?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1790** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1790?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#1797](https://github.com/terraware/terraware-server/pull/1797) <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1797?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @sgrimm and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/test/kotlin/com/terraformation/backend/accelerator/db/AcceleratorOrganizationStoreTest.kt [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      insertOrganizationInternalTag(acceleratorOrgId1, InternalTagIds.Accelerator)
+      insertOrganizationInternalTag(acceleratorOrgId2, InternalTagIds.Accelerator)
+      insertOrganizationInternalTag(nonAcceleratorOrgId, InternalTagIds.Internal)
+
+      insertProject(organizationId = nonAcceleratorOrgId)
+      insertProject(organizationId = untaggedOrgId)
+      insertProject(organizationId = 1, participantId = participantId)
+      val unassignedProjectId1 = insertProject(organizationId = 1, name = "A")
+      val unassignedProjectId2 = insertProject(organizationId = 2, name = "C")
+      val unassignedProjectId3 = insertProject(organizationId = 2, name = "B")
```
Comment (karthikbtf) [inline src/test/kotlin/com/terraformation/backend/accelerator/db/AcceleratorOrganizationStoreTest.kt:51]: Use the `acceleratorOrgId{1,2}` consts here and in the test below?

**Review by karthikbtf — APPROVED** [review]
Looks great to me, thanks so much 🙏

**Comment by karthikbtf** [comment]
@sgrimm just occurred to me - since the response payload includes project participant id, the client can filter out projects using the participant id as criteria. In case it helps simplify the BE code and sorry about the extra work. We can go either way, just wanted to call out this option.

**Comment by sgrimm** [comment]
I did it this way because as the list of cohorts grows over time, we'd be returning more and more irrelevant results in the API response.

**Comment by karthikbtf** [comment]
> I did it this way because as the list of cohorts grows over time, we'd be returning more and more irrelevant results in the API response.

Got it, makes sense.

**Pull request merged**